### ISSUE TITLE
[Feature Request] scheduling requests while sending them between lotus-workers in a miner actor

### ISSUE THREAD

**Issue by sherviniv** [filecoin-project/lotus#6080]
![alt text](https://i.ibb.co/r2pqmNX/index.jpg)
[You can find the image here](https://ibb.co/B26vtFY)
There is a Lotus worker that does some processes on files. Assume that multiple System-A workers are working independently at the same time. Once their job is finished, they both send result files to another worker type, called System-B. Assume sending files from a System-A worker to a System-B worker takes around 20 minutes. if two System-A workers try to send their files to System-B workers at the same time, due to the network capacity, it will take up to 40 minutes. As a result, System-B needs to wait for a longer time to receive files from a specific System-A worker.
I'm looking for a way that we can schedule the jobs in a clever way and change scheduler configs. 
the data transfer between step PC1 and PC2 is ~500GB. if you have two boxes that both do PC1 and PC2. there are times it transfers both directions at the same time. the only problem is PC1 + PC2 on more than one machine

Thanks

**Comment by Reiers** [comment]
Hi @sherviniv 

Take a look at this:
https://lotus.filecoin.io/docs/storage-providers/seal-workers/#sector-storage-groups

I think that will solve the issue you are encountering. 

If you are still experiencing problems, open a new ticket or ask for help in our [Discussions forum.](https://github.com/filecoin-project/lotus/discussions)

Thanks for the issue ticket!

**Comment by sherviniv** [comment]
@Reiers Yep, That's all was needed, Thanks
